src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -201,18 +209,47 @@ def _migrate_external_table(self, src_table: Table, rule: Rule, grants: list[Gra
         self._backend.execute(src_table.sql_alter_from(rule.as_uc_table_key, self._ws.get_workspace_id()))
         return self._migrate_acl(src_table, rule, grants)
 
-    def _migrate_dbfs_root_table(self, src_table: Table, rule: Rule, grants: list[Grant] | None = None):
-        target_table_key = rule.as_uc_table_key
-        table_migrate_sql = src_table.sql_migrate_dbfs(target_table_key)
-        logger.debug(f"Migrating managed table {src_table.key} to using SQL query: {table_migrate_sql}")
+    def _migrate_external_table_hiveserde(
+        self, src_table: Table, rule: Rule, grants: list[Grant] | None = None, mounts: Iterable[Mount] | None = None
+    ):
+        if not self._hiveserde_in_place_migrate:
```

Comment:
propagate this as an argument, as you already propagate the `mounts`.

Reply:
why do we even need this property, if we explicitly supply `migrate_tables(what=What.EXTERNAL_HIVESERDE, ...)`?..